pylintrc

```diff
@@ -326,7 +326,7 @@ max-attributes=7
 min-public-methods=2
 
 # Maximum number of public methods for a class (see R0904).
-max-public-methods=22
+max-public-methods=25
```

Comment:
@antoviaque We already bumped that from 20 to 22 a while ago, so I thought why not bump it a bit more :) It might be better to disable the warning on the class instead, we already do that for a related warning: https://github.com/open-craft/opencraft/blob/53e65a3f3926f9f6e9ca17715eb6dea0e0cfd33d/instance/models/instance.py#L313

The three methods that I added are very simple, there's no logic in them, and they mirror similar methods for studio subdomains. I don't think code quality would be any better if they would be split out into a a mixin for no other reason than to keep the linter happy.